ibm/service/cis/resource_ibm_cis_ruleset.go

```diff
@@ -112,7 +112,7 @@ var CISResourceResponseObject = &schema.Resource{
 												Description: "Rules",
 												Elem: &schema.Resource{
 													Schema: map[string]*schema.Schema{
-														CISRulesetsId: {
+														CISRulesetRuleId: {
```

Comment:
Is this a breaking change we are renaming an attribute pls follow guidelines
https://developer.hashicorp.com/terraform/plugin/framework/deprecations